### PR TITLE
doc: Add subscription process of TTSC via AWS Marketplace

### PR DESCRIPTION
**Pull request by Siva-7482**
#### Summary

Process for subscribing to The Things Stack Cloud via AWS Marketplace.
Ref: https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/430

#### Screenshots

...

#### Changes

- Added a new section with the process of subscribing to The Things Stack Cloud via AWS Marketplace.
- Added the relevant screenshots.

#### Notes for Reviewers

@KrishnaIyer, please review.

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Review by adriansmares — COMMENTED** [review]
This LGTM but these `:Zone.Identifier` files shouldn't be committed.

**Comment by Siva-7482** [comment]
Sure, @adriansmares. The zone identifier files were removed and updated the PR.

**Review by adriansmares — APPROVED** [review]

File: doc/content/the-things-stack/cloud/subscribe-tts-via-aws-marketplace/_index.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+weight: 2
+---
+
+This section shows how to subscribe a tenant in {{% tts %}} Cloud deployment via AWS Marketplace.
+
+<!--more-->
+
+## Subscribing {{% tts %}} Cloud tenant
+
+Navigate to [The Things Stack Cloud for LoRaWAN](https://aws.amazon.com/marketplace/pp/prodview-djx65iyt6w5cy) in AWS Marketplace.
```
Comment (nejraselimovic) [inline doc/content/the-things-stack/cloud/subscribe-tts-via-aws-marketplace/_index.md:13]:
```suggestion
Navigate to [{{% tts %}} Cloud for LoRaWAN](https://aws.amazon.com/marketplace/pp/prodview-djx65iyt6w5cy) in AWS Marketplace.
```

File: doc/content/the-things-stack/cloud/subscribe-tts-via-aws-marketplace/_index.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+{{< figure src="setup-product.png" alt="Set up product" >}}
+
+You will observe a pop up, i.e., `To continue, set up your account and complete the registration`. Then Click on `Set up your account`.
+
+{{< figure src="setup-account.png" alt="Set up your account" >}}
+
+Fill in all the details on the `Configure your account` page.
+
+{{< figure src="configure-account.png" alt="Configure Account" >}}
```
Comment (nejraselimovic) [inline doc/content/the-things-stack/cloud/subscribe-tts-via-aws-marketplace/_index.md:37]: Can you please retake screenshots in a way described in [CONTRIBUTING doc](https://github.com/TheThingsIndustries/lorawan-stack-docs/blob/master/CONTRIBUTING.md)? This one looks off, and the `setup-product.png` above is unequally cropped

**Review by nejraselimovic — CHANGES_REQUESTED** [review]

**Review by nejraselimovic — APPROVED** [review]

**Pull request merged**